server/sqlflowserver_test.go

```diff
@@ -143,6 +143,19 @@ func TestGoroutineLeaky(t *testing.T) {
 	}
 }
 
+func TestSessionDB(t *testing.T) {
```

Comment:
maybe also need to test sending session message throught RPC

Reply:
done.